README.md

```diff
@@ -60,7 +60,7 @@ cd EE_API_automation/pyresttest/
 ./run_forge_import_api_test.sh YOUR_OSIO_ACOUNT YOUR_OSIO_TOKEN
 ```
 
-> NOTE: Depending wether your github account belongto an organisation or not the API response is different. the bash script wraps the differentce and run the relevant tests.
+> NOTE: Depending whether your GitHub account belong to an organisation or not the API response is different. the bash script wraps the difference and run the relevant tests.
```

Comment:
belong**s** to
**T**he bash 
run**s** the

Reply:
again, thank you!